### DISCUSSION TITLE
识别出来经常会漏掉一位该怎么处理呢？

### DISCUSSION THREAD

**Discussion by 0-Maxwei-0** [PaddlePaddle/PaddleOCR#12999] [Q&A]
识别一些英文数字组合的图片，识别结果经常会漏掉一个或者几个字符，应该如何调整？

**Comment by thsno02** [comment]
框外延一些试试，这个最简单。

**Comment by 0-Maxwei-0** [comment]
> 框外延一些试试，这个最简单。

可以修改哪个参数实现呢？我尝试修改了unclip_ratio但效果不佳

**Comment by thsno02** [comment]
有一个 dilation 的参数，调大试试。或者你手动截屏也行，先试试外延 work 不 work。
